tasks/managed/push-snapshot/push-snapshot.yaml

```
@@ -196,7 +196,8 @@ spec:
               if [[ "$COPY_ATTACHED_ARTIFACTS" == "true" && "${artifact_count}" -gt 0 ]]; then
                 # Copy the image and all attached artifacts
                 oras cp -r \
-                  --registry-config "$DOCKER_CONFIG/config.json" \
+                  --from-registry-config "$DOCKER_CONFIG/config.json" \
+                  --to-registry-config "$DOCKER_CONFIG/config.json" \
```

Wouldn't this be more suitable?
```suggestion
                  --from-registry-config "$SOURCE_AUTH_FILE" \
                  --to-registry-config "$DEST_AUTH_FILE" \
```

---

Ah, I completely missed that. Thanks for the suggestion @mmalina , fixed.